impl/maven-impl/src/test/java/org/apache/maven/impl/DefaultPluginXmlFactoryTest.java

```diff
@@ -94,8 +95,8 @@ void readFromPathParsesPluginDescriptorCorrectly() throws Exception {
         Path xmlFile = tempDir.resolve("plugin.xml");
         Files.write(xmlFile, SAMPLE_PLUGIN_XML.getBytes());
         assertThat(defaultPluginXmlFactory
-                        .read(XmlReaderRequest.builder().path(xmlFile).build())
-                        .getName())
```

Comment:
bug in formation setup (SPOT), diff is not acceptable.
- https://github.com/apache/maven/pull/2322